app/models/manageiq/providers/redhat/infra_manager/api_integration.rb

```diff
@@ -237,6 +238,7 @@ def verify_credentials(args)
         :ca_certs         => ca_certs,
         :metrics_username => metrics_username,
         :metrics_password => ManageIQ::Password.try_decrypt(metrics_password),
+        :metrics_server   => metrics_server,
```

Comment:
We also never even passed the metrics_server in, so honestly not sure how this ever worked